cmd/bootstrap/transit/.gitignore

```diff
@@ -3,3 +3,4 @@
 *.enc
 transit
 demo/
+old
```

Comment:
What is this for?

Reply:
I moved the older scripts to the folder to view but forgot to remove the `.gitignore`. Removing...